sql/core/src/test/resources/sql-tests/inputs/change-column.sql

```diff
@@ -2,54 +2,38 @@
 CREATE TABLE test_change(a INT, b STRING, c INT) using parquet;
 DESC test_change;
 
--- Change column name (not supported yet)
```

Comment:
do we have a test file for RENAME COLUMN? If not can we add some tests here to keep the test coverage?

Reply:
ok.